scene/3d/lightmap_gi.cpp

```diff
@@ -1352,6 +1365,7 @@ void LightmapGI::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_POST_ENTER_TREE: {
 			if (light_data.is_valid()) {
+				ERR_FAIL_COND_MSG(light_data->is_using_spherical_harmonics() && !light_data->is_using_packed_directional(), "The directional lightmap textures are stored in a format that isn't supported anymore. Please rebake to fix the issue.");
```

Comment:
Since the packing is implemented as a separate pass, could we not just run that pass when we encounter an old lightmap and upgrade it automatically for the user?

Reply:
I thought about it, but older lightmaps wouldn't work the same way since they use different coefficients

Reply:
Oh right, I guess https://github.com/godotengine/godot/pull/95888 technically breaks compatibility already for existing lightmaps